File: web/src/model/victron_controller/dashboard/from_0_2_0_wsservermessage_pong.ts
// @ts-nocheck
import {Pong as v0_2_0_Pong} from './v0_2_0/WsServerMessage'
import {Pong as dashboard_Pong} from './WsServerMessage'

export function convert__ws_server_message__pong__from__0_2_0(from: v0_2_0_Pong): dashboard_Pong {
    return new dashboard_Pong (
        JSON.parse(JSON.stringify(from.body))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_wsservermessage_pong.rs


pub fn convert__wsservermessage__pong__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::ws_server_message::Pong) -> crate::victron_controller::dashboard::ws_server_message::Pong {
    crate::victron_controller::dashboard::ws_server_message::Pong {
        body: serde_json::from_value(serde_json::to_value(&from.body).unwrap()).unwrap(),
    }
}